source/app/web/statics/about/index.html

```diff
@@ -190,7 +190,7 @@ <h2>
                       <svg v-else xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="16" height="16"><path fill-rule="evenodd" d="M2.75 2.5a.25.25 0 00-.25.25v7.5c0 .138.112.25.25.25h2a.75.75 0 01.75.75v2.19l2.72-2.72a.75.75 0 01.53-.22h4.5a.25.25 0 00.25-.25v-7.5a.25.25 0 00-.25-.25H2.75zM1 2.75C1 1.784 1.784 1 2.75 1h10.5c.966 0 1.75.784 1.75 1.75v7.5A1.75 1.75 0 0113.25 12H9.06l-2.573 2.573A1.457 1.457 0 014 13.543V12H2.75A1.75 1.75 0 011 10.25v-7.5z"></path></svg>
                       <div class="content">
                         Commented on <a :href="`https://github.com/${repo}/${{issue:'issues', pr:'pull', commit:'commit'}[event.on]}/${event.number}`">#{{ event.number }} {{ event.title }}</a> from <a :href="`https://github.com/${repo}`">{{ repo }}</a>
-                        <quote v-if="event.content.trim().length" v-html="event.content"></quote>
+                        <quote v-if="event.content.trim().length" v-html="format('comment', event.content, {repo})"></quote>
                       </div>
```

Comment:
It could be revelant to add it to issues and prs (i.e. first comments) too :
https://github.com/lowlighter/metrics/blob/553a1a1c3823201c20f74db237f018eb996e90f8/source/app/web/statics/about/index.html#L230
https://github.com/lowlighter/metrics/blob/553a1a1c3823201c20f74db237f018eb996e90f8/source/app/web/statics/about/index.html#L237